modules/internal/ChapelBase.chpl

```diff
@@ -956,18 +968,21 @@ module ChapelBase {
     }
   }
 
-  proc _upEndCount(param countRunningTasks=true) {
-    var e = __primitive("get end count");
+  proc _upDynamicEndCount(param countRunningTasks=true) {
+    var e = __primitive("get dynamic end count");
     _upEndCount(e, countRunningTasks);
   }
 
-  proc _downEndCount() {
-    var e = __primitive("get end count");
+  // MPF - should this have
+  //pragma "dont disable remote value forwarding"
```

Comment:
Not a well-informed or strong one. I'd be inclined to add the pragma and do some testing as our existing downEndCount has the pragma.

I'm unfamiliar with the historical motivating use-cases for this pragma or what impact is has, so I'm not too sure one way or another.